app/helpers/application_helper.rb

```diff
@@ -1,2 +1,20 @@
 module ApplicationHelper
+  def sort_by(column, label: nil)
+    label ||= column
```

Comment:
```suggestion
    label ||= column.capitalize
```
にするとほとんどの場合でタイトル引数が省略できて、かっこいいと思います。

Reply:
修正しました。
updated_atのリンクだけはlabel指定を省略すると表示としてUpdated_atとアンダーバーが入ってしまうため、updated_atのみTitle指定を残す形にしました。
https://github.com/yush-nh/annotation_guidelines/pull/52/commits/3d7ea79d9607ea50cad266fd566753e7462c3aba